extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -491,4 +496,19 @@ private boolean isEqual(State state1, State state2) {
         return colorModeIsEqual && effectIsEqual;
     }
 
+    @Override
+    public Collection<ConfigStatusMessage> getConfigStatus() {
+        // The bridge IP address to be used for checks
+        final String bridgeIpAddress = (String) getThing().getConfiguration().get(HOST);
+
+        // Check whether an IP address is provided
+        if (bridgeIpAddress == null) {
+
+            return ImmutableList.of(ConfigStatusMessage.Builder.error(HOST)
```

Comment:
To me it makes sense, since `Collections.singletonList(o)` is already in use for the class and there is no need to make use of _com.google.common.collect.ImmutableList_. @tomhoefer, what is your opinion on the subject, since we previously had a discussion and you insisted that I introduce the use of Guava and `ImmutableList.of()`?